packages/spatial/src/transform/components/EntityTree.test.tsx

```diff
@@ -44,9 +46,32 @@ import {
   removeFromEntityTree,
   traverseEntityNode,
   traverseEntityNodeParent,
+  useAncestorWithComponent,
+  useChildWithComponent,
   useTreeQuery
 } from './EntityTree'
 
+const Validate = {
```

Comment:
let's just make this a function `assertEntityTree`

Reply:
Changed with 1fce025